amundsen_application/static/js/components/DashboardPage/ImagePreview/index.tsx

```diff
@@ -58,7 +58,7 @@ export class ImagePreview extends React.Component<ImagePreviewProps, ImagePrevie
         }
         {
           this.state.hasError &&
-          <Linkify className='body-placeholder'>{`${Constants.ERROR_MESSAGE} ${redirectUrl}`}</Linkify>
+          <Linkify className='body-placeholder' properties={{ target: '_blank' }}>{`${Constants.ERROR_MESSAGE} ${redirectUrl}`}</Linkify>
```

Comment:
It is a good practice to add `rel="noopener noreferrer"` along with the target: '_blank' for performance and security reasons.

You can [read more here.](https://web.dev/external-anchors-use-rel-noopener/)

Reply:
Okay, I will change it. Thank you!